pyproject.toml

```diff
@@ -1,7 +1,55 @@
 [build-system]
-requires = ["setuptools", "lsst-versions"]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "lsst-felis"
+description = "Felis is a way of describing database catalogs, scientific and otherwise, in a language and DBMS agnostic way."
+license = {text = "BSD 3-Clause License"}
```

Comment:
felis is currently GPLv3 I think.

Reply:
Changed to:

`license = {text = "GNU General Public License v3 (GPLv3)"}`

and also modified the classifiers section.